manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/persistence/models/Bridge.java

```diff
@@ -13,13 +13,28 @@
 import javax.persistence.Table;
 import javax.persistence.UniqueConstraint;
 
+import org.hibernate.annotations.Filter;
+import org.hibernate.annotations.FilterDef;
+import org.hibernate.annotations.FilterDefs;
+import org.hibernate.annotations.Filters;
+import org.hibernate.annotations.ParamDef;
+
 @NamedQueries({
         @NamedQuery(name = "BRIDGE_V2.findByNameAndCustomerId",
                 query = "from Bridge_V2 where name=:name and customer_id=:customerId"),
         @NamedQuery(name = "BRIDGE_V2.findByIdWithConditions",
                 query = "from Bridge_V2 b left join fetch b.conditions where b.id=:id"),
         @NamedQuery(name = "BRIDGE_V2.countByOrganisationId",
-                query = "select count(*) from Bridge_V2 where organisation_id=:organisationId")
+                query = "select count(*) from Bridge_V2 where organisation_id=:organisationId"),
+        @NamedQuery(name = "BRIDGE_V2.findByCustomerId",
+                // This query returns duplicated items!
```

Comment:
It should be possible to do `select distinct(b)` directly in the query [similar to this](https://github.com/lucamolteni/sandbox/blob/6c5234166063734d286141bb587d1ece9a8916cc/manager/src/main/java/com/redhat/service/smartevents/manager/models/Processor.java#L35).